pkg/auth/authz_handler.go

```diff
@@ -162,6 +269,11 @@ func (a *AuthzHandler) allowedToUseAssistedInstaller(username string) (bool, err
 		context.Background(), username, ocm.AMSActionCreate, ocm.BareMetalClusterResource)
 }
 
+func (a *AuthzHandler) hasClusterEditRole(payload *ocm.AuthPayload, subscriptionID string) (bool, error) {
```

Comment:
can we pass subscriptionID as UUID instead of string? should be nicer to get the string value when invoking the AccessReview API.

Reply:
All the APIs in this file are using strings.